### DISCUSSION TITLE
IIP demurrage Idea - introduce limited supply

### DISCUSSION THREAD

**Discussion by ubiubi18** [idena-network/idena-docs#177] [IIPs]
This is an idea how to get away from unlimited supply and incentify more active participation at the same time. https://github.com/ubiubi18/IIP-demurrage

**Comment by morienzo** [comment]
The creation of IDNA coins is already a tax, as it erodes the existing coins. Therefore, I do not understand the point of implementing this mechanism with a finite number, except to please investors. I am against the idea of a global tax, as it would further disadvantage those who cannot mine.

**Reply by morienzo** [reply]
Instead of having a tax on what is earned, wouldn't it be interesting to reduce or increase inflation based on certain criteria? For example, issuing IDNA coins based on network usage or zero wallet size.

**Comment by reb0rn21** [comment]
This is crypto you can not take tax from balance.... investors all left with this 1-2 left will leave also 
My idea would be to issue coins to mining/validation vs number of active identity, it does not need to be linear issuance but less ppl validating less inflation

**Reply by ubiubi18** [reply]
Look here at 1:13:30, another crypto billionaere proposing a 7% taxaction effect per year wirh demurrage, which is kind of the same, just upscalable by the numbers of identities joining the system.  https://m.youtube.com/live/xRAG5oIrwH4?si=e7ebjCuULkwDV1rY&t=4410
